crashlytics/src/IOSImpl.cs

```diff
@@ -116,6 +117,19 @@ public override void LogException(Exception exception)
     private void RecordCustomException(LoggedException loggedException, bool isOnDemand) {
       Dictionary<string, string>[] parsedStackTrace = loggedException.ParsedStackTrace;
 
+      if (isOnDemand && parsedStackTrace.Length == 0) {
+        // if for some reason we don't get stack trace from exception, we add current stack trace in
+        var currentStackTrace = System.Environment.StackTrace;
+        LoggedException loggedExceptionWithCurrentStackTrace = new LoggedException(loggedException.Name, loggedException.Message, currentStackTrace);
+        parsedStackTrace = loggedExceptionWithCurrentStackTrace.ParsedStackTrace;
+
+        if (parsedStackTrace.Length > 2) {
```

Comment:
If we don't add this logic, the stack trace will look like this as it still blame on crashlytics frame:
<img width="1229" alt="Screen Shot 2023-01-13 at 12 27 19 AM" src="https://user-images.githubusercontent.com/16548721/212244244-1fee9d5d-c0cf-4e4f-856b-21bc765d4532.png">

Remove these two frame for logging purpose then here is the result:
<img width="1122" alt="Screen Shot 2023-01-13 at 12 26 43 AM" src="https://user-images.githubusercontent.com/16548721/212244227-42c21b1f-72a0-4ee6-9bae-99f3f6be9eaf.png">